### ISSUE TITLE
pretrained models

### ISSUE THREAD

**Issue by kiyse0713** [qidiso/mobilefacenet-V2#4]
Could you please upload your highest accuracy pretrained models? @qidiso

**Comment by qidiso** [comment]
because the training is not end, if i finished ,i will

**Comment by qidiso** [comment]
you can visit my log file:
[https://github.com/qidiso/mobilefacenet-V2/blob/master/wd1e-6-0.01.log](https://github.com/qidiso/mobilefacenet-V2/blob/master/wd1e-6-0.01.log)

**Comment by kiyse0713** [comment]
您好，我是按照您下面的网址的pipeline进行训练的，第一步loss是softmax，lr是0.1，第二步和第三步的loss是arcface，但是第二部的pretrained models 和 第三步的 pretrained models是不是顺序换了，第二步都还没有model-y1-arcface,80

https://github.com/qidiso/mobilefacenet-mxnet





At 2018-07-30 16:56:59, "qidiso" <notifications@github.com> wrote:


you can visit my log file:
https://github.com/qidiso/mobilefacenet-V2/blob/master/wd1e-6-0.01.log

**Comment by qidiso** [comment]
I suggest you take second steps to train with s=128, and the third step is to train with s=64, which will make the value of LFW higher.The second step is not model-y1-arcface, 80，you can take the model-y1-sofmax,80 instead. i training order is : softmax,s=64 ,s=128,s=64.this make the lfw higher.

**Comment by chenkingwen** [comment]
what's the hyper-parameters of the first step by using softmax to train the net reaching the accuracy 99.2% on lfw? thank you very much.

**Comment by chenkingwen** [comment]
what's the hyper-parameters of the first step by using softmax to train the net reaching the accuracy 99.2% on lfw? thank you very much.@qidiso

**Comment by ramanathan831** [comment]
@qidiso Can you provide the exact commands of each steps in the ReadMe file please?
